data/config.yaml

```diff
@@ -1,8 +1,8 @@
 _id: d779730c7cc94155877a24e2e8be2ad1
-created: '2022-04-18T14:36:30.754421'
+created: '2022-09-09T13:05:49.279830'
```

Comment:
This file should not get checked in for PRs - running the tests keep changing this for some reason, and we haven't managed to fix that yet.

Reply:
So is there a way to remove this file from the PR or were you just noting that, ideally, changes in this file shouldn't be picked up by PRs?

Reply:
There are ways to revert changes to a file after a commit has been made (git can do just about anything). @lbianchi-lbl is probably the best person to ask on how to do it (there are many ways it can be done, but there are different side-effects).

Reply:
Note however- this file is meant to be there, it just should not be changed by a PR (i.e. you want to revert your changes, not delete the file).

Reply:
> There are ways to revert changes to a file after a commit has been made (git can do just about anything). @lbianchi-lbl is probably the best person to ask on how to do it (there are many ways it can be done, but there are different side-effects).

The following should work (where 0309eec is the last commit on `main` off of which this branch was created):

```sh
git checkout 0309eec -- data/config.yaml
git commit -m "Revert changes to data/config.yaml"
```